servers/zms/src/test/java/com/yahoo/athenz/zms/provider/ServiceProviderManagerTest.java

```diff
@@ -64,7 +62,7 @@ public void clean() {
     public void testIsServiceProvider() throws InterruptedException, ExecutionException {
         String testDomainName = "test.domain";
         String testRoleName = "test_role";
-        long fetchFrequency = 1L; // For the test, fetch every second
+        long fetchFrequency = 5L; // For the test, fetch every second
```

Comment:
Should we update the comment accordingly?

Reply:
Yes, thanks.updated the comment